docs/design/domain-class-types-in-kotlin-sdk.md

```diff
@@ -29,140 +29,164 @@ structure GetCityInput {
 }
 ```
 
-### General class definition
+### Option - Class Sample
+
+NOTE: Some ancillary generated functions have been removed from the following snippets to keep focus.
 
 ```kotlin
-class GetCityInput private constructor(builder: BuilderImpl) {
-    val cityId: String? = builder.cityId
+class GetCityRequest private constructor(builder: BuilderImpl) {
```

Comment:
**Comment**: If this is going to change to `GetCityRequest` in code then the test of the doc's references to `GetCityInput` should be updated too.

Reply:
The issue that led to this PR was specifically about the clarity and consistency of the documentation. If we exchange `GetCityInput`/`GetCityImageInput` confusion for `GetCityInput`/`GetCityRequest` confusion, we'll be in no better shape. I think if we want to show shape renaming, that should probably be confined to an appendix, subsection, etc. rather than blended with the main thesis of the doc.

Reply:
Before this PR there was just a random mixing of two different modeled types in description and code snippets.  Now, there is no mixing.  The model type snippets are directly from the canonical Smithy example.  The codegen is directly from what our SDK would produce.  To me this follows the principle of least surprise, and allows the reader to reproduce the results if they care to.